src/rgw/rgw_admin.cc

```diff
@@ -1873,7 +1873,7 @@ stringstream& push_ss(stringstream& ss, list<string>& l, int tab = 0)
 static void get_md_sync_status(list<string>& status)
 {
   RGWMetaSyncStatusManager sync(store, store->svc()->rados->get_async_processor());
-
+  RGWOp_MDlog_CompareStatus meta;
```

Comment:
RGWOp_MDlog_CompareStatus implements a REST api. you can't call it directly here, because you want it to execute on the _source_ zone. you have to actually send an http request to the source zone

there's a decent example of sending an http request to an /admin/log api in RGWRemoteDataLog::read_log_info()